design/schedule-unpaused-triggers-config_design.md

```diff
@@ -0,0 +1,113 @@
+# Schedule Unpaused Triggers Config Design
+## Abstract
+User want to configure what happens when schedule is unpaused. Either run backup immediately or at the next cron schedule.  This design proposes to add a new field to the schedule spec to allow user to configure the behavior
+
+## Background
+Currently, the default behavior is to run the trigger immediately and may not be desired by all users (https://github.com/vmware-tanzu/velero/issues/6517)
+
+## Goals
+- Make schedule unpaused behavior configurable
+
+## Non Goals
+- Changing the default behavior
+
```

Comment:
Maybe also add a use-case/scenario/user story section, one example would suffice I think.

Reply:
It's already in the background section. Will expand on it.

Dedicated use-case/scenario section is not in the design template.